test/e2e/metrics_e2e_test.go

```diff
@@ -261,6 +266,10 @@ func getPodWithLabel(client operatorclient.ClientInterface, label string) *corev
 	return &podList.Items[0]
 }
 
+func InstallPlanApproval(ip v1alpha1.Approval) string {
```

Comment:
`InstallPlanApproval` isn't too descriptive. Function name should always be a verb/define an action. Example: `InstallPlanApprovalToString`

Reply:
Also, function names staring with capital letters are used to indicate exported functions. Since we don't want to export this function outside this package, `installPlanApprovalToString` seems more appropriate.

Reply:
Thanks for the feedback! I changed the func name to follow the coding style.